### ISSUE TITLE
Slightly confusing error message when there are two devices

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#578]
With both an emulator running on my machine, and a phone plugged in, when I try to run my app, I get:

```
[myapp/] flutter start
Warning: Multiple Android devices are connected, but no device ID was specified.
severe: Unable to run application - no connected devices.
• exited with code 2 •
```

The first warning makes sense. However, the second error is a bit confusing. "no connected devices" might be better as "please choose a specific target device"

Thoughts?

Thanks!

**Comment by abarth** [comment]
Yeah, the second message is generic in the code and doesn't understand what really happened.
